packages/api/src/objectSet/ObjectSetListener.ts

```diff
@@ -45,7 +45,7 @@ export interface ObjectSetListener<
   /**
    * There was a fatal error with the subscription process. The subscription will close or will not be established.
    */
-  onError?: (errors: Array<any>) => void;
+  onError?: (errors: { subscriptionClosed: boolean; error: any }) => void;
```

Comment:
Settled on a boolean here, the alternative was an enum to also be able to display the "I already tried to reconnect for you case" but I think that adds unnecessary complexity

Reply:
maybe I need to make it isSubscriptionClosed?